driver.py

```diff
@@ -103,8 +103,8 @@ def collect_experience(self, record=1, vis=0, n_steps=None, noise_flag=True, sta
             observation = self.env.reset()
             observation = observation['image']
         else:
-            qposs, qvels = alg.er_expert.sample()[5:]
-            observation = self.env.reset(qpos=qposs[0], qvel=qvels[0])
+            states, actions, rewards, posstates, terminals = alg.er_expert.sample()
+            observation = states[5]
```

Comment:
Why are we picking particularly the 5th entry of the states?

Reply:
it seems like the old code picks the 5th entry too here `alg.er_expert.sample()[5:]`? I assumed because the first few states maybe unreliable somehow?
Should i change it to 0?

Reply:
the old code `[5:]` is just picking the values starting from the 5th entry which are originally qpos and qvels

Reply:
yes, I think states[0] should be fine, it just takes the 1st state in the original code, just it recovers it from qpos and qvel